library-and-framework-list.json

```diff
@@ -138,5 +138,537 @@
         "test_level": "fully-tested"
       }
     ]
+  },
+  {
+    "artifact": "org.springframework.boot:spring-boot-starter-actuator",
+    "description": "Starter for using Spring Boot's Actuator which provides production ready features to help you monitor and manage your application",
+    "details": [
+      {
+        "minimum_version": "3.0.0",
+        "metadata_locations": [],
```

Comment:
Why the metadata locations are not provided here? I believe we need them, since our schema requires it.

Reply:
I've just updated the PR to add `"medata_locations"` entries. I've also rebased on top of the current master.